scorecard/tests/indicators/test_uifw_expenditure.py

```diff
@@ -54,13 +54,13 @@ def test_result(self):
                         "date": 2019,
                         "result": 2.68,
                         "rating": "bad",
-                        "cube_version": "v2"
+                        "cube_version": "v1"
```

Comment:
let's bump this indicator to 2020 as well so we have at least some coverage of its v2 functionality